src/main/resources/translations.json

```
@@ -170,7 +171,8 @@
       "somethingWentWrong": "Something went wrong",
       "invalidDifficulty": "The difficulty has to be between 0 and 10",
       "anvilOutput": "Click here to accept",
-      "noNameSet": "No name set"
+      "noNameSet": "No name set",
+      "enchantNotEnabled": "This enchantment is not activated on this server."
```

one time 'this' less
```suggestion
      "enchantNotEnabled": "This enchantment is currently not activated."
```